modules/apps/object/object-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Object API
 Bundle-SymbolicName: com.liferay.object.api
-Bundle-Version: 44.0.1
+Bundle-Version: 44.1.0
```

Comment:
hey @p-albuquerque could you please move this commit to after `LPS-165576 Create BusinessType for MultiSelectPicklist Field`?